HerPublicWebsite.UnitTests/BusinessLogic/Services/SessionRecorderServiceTests.cs

```diff
@@ -40,4 +41,37 @@ public async Task RecordNewSessionStarted_WhenCalled_PersistsASessionWithCorrect
         mockDataAccessProvider.Verify(dap => dap.PersistSession(It.Is<Session>(s => s.Timestamp == now)), Times.Once);
         mockDataAccessProvider.VerifyNoOtherCalls();
     }
+
+    [Test]
+    public async Task
+        SetIsJourneyCompleteToTrue_WhenCalledWithAQuestionnaireWithASessionId_CallsDataAccessProvidersSetIsJourneyCompleteToTrue()
+    {
+        // Arrange
+        const int sessionId = 2;
+        var questionnaire = QuestionnaireHelper.InitializeQuestionnaire();
+        questionnaire.SessionId = 2;
```

Comment:
could sessionId const be used here?

Reply:
...Yes, that was why I put it in initially. Thank you for spotting!